scripts/check_layer_size.sh

```diff
@@ -7,10 +7,9 @@
 
 # Compares layer size to threshold, and fails if below that threshold
 
-# 7 mb size limit
 set -e
-MAX_LAYER_COMPRESSED_SIZE_KB=$(expr 4 \* 1024)
-MAX_LAYER_UNCOMPRESSED_SIZE_KB=$(expr 13 \* 1024)
+MAX_LAYER_COMPRESSED_SIZE_KB=$(expr 5 \* 1024)
+MAX_LAYER_UNCOMPRESSED_SIZE_KB=$(expr 12 \* 1024)
```

Comment:
Bytecode is harder to compress (hence increase from 4->5) but uncompressed is much smaller (hence decrease from 13->12)

Reply:
Sounds good! Could we update this changes in `serverless-tools` once this is merged? Thanks!